tests/azure_iot/testdata/azure_iot.txt

```diff
@@ -0,0 +1,37 @@
+[!exec:bash] stop
+[!exec:grep] stop
+[!exec:cat] stop
+[!exec:curl] stop
+
+exec docker run  --name azure_iot  dautovri/eden-azure-iot-test
+exec docker cp  azure_iot:/disk  disk/
+exec docker rm -f azure_iot
+
+eden -t 10m pod deploy -p 8028:80 --vnc-display=1 file://disk/Ubuntu-18.04-bionic --disks=file://disk/Ubuntu-18.04-bionic-1 --disks=file://disk/Ubuntu-18.04-bionic-2 --disks=file://disk/Ubuntu-18.04-bionic-3 -n azureiot --memory 3GB -v debug
+stdout 'deploy pod azureiot with ubuntu qcow2 disks request sent'
```

Comment:
Here we should use something like `stdout 'deploy pod azureiot.*request sent'`. Current variant not works for me.

Reply:
Changed, on  `test eden.app.test -test.v -timewait 20m RUNNING azureiot`